docs/source/tutorial.rst

```diff
@@ -0,0 +1,126 @@
+Tutorial
+--------
+
+For this tutorial we are going to use a well known case file from `NIST <https://cfreds.nist.gov/all/NIST/HackingCase>`_.
+In this tutorial we assume Linux is used, on other operating systems most steps are the same but details may differ.
+First of all, we install Python and create a venv:
+
+.. code-block:: console
+
+    $ python3 -m venv dissect
+
+Now activate the virtual environment to get some work done:
+
+
+.. code-block:: console
+
+    $ source dissect/bin/activate
```

Comment:
You never install dissect using pip.

Reply:
To be completely honest, writing these steps out seems a bit overkill. You can in my opinion redirect them to the "install" page of dissect. This page also describes the requirements/prerequisites of dissect and this page doesn't.

Reply:
> You never install dissect using pip.

This is done on the install page, which comes before the tutorial.

Reply:
> To be completely honest, writing these steps out seems a bit overkill. You can in my opinion redirect them to the "install" page of dissect. The install page also describes the requirements/prerequisites of dissect and this page doesn't.

Not everyone knows how to use Python. Redirecting to other sources might turn people away. Dissect is not just about Python, it's an IR tool. We might attract an audience that has a different skill set and appreciates the extra steps.